### ISSUE TITLE
okhttp crumbs are duplicated

### ISSUE THREAD

**Issue by mrzbn** [getsentry/sentry-android-gradle-plugin#617]
### Gradle Version

7.5

### AGP Version

7.4.2

### Code Minifier/Optimizer

R8

### Version

4.1.0

### Sentry SDK Version

7.1.0

### Steps to Reproduce

1. add dependency and plugin
2. throw an Exception
3. okhttp crumbs are duplicated

### Expected Result

not showing duplicate crumbs.

also crumbs are supposed to be sorted. but they are not. (screenshot attached)

### Actual Result

![Screenshot 2023-12-23 143436](https://github.com/getsentry/sentry-android-gradle-plugin/assets/58802944/7255594d-1478-4348-add0-1d889a8592e9)

**Comment by markushi** [comment]
Hi @mrzbn, thanks for reaching out!

Could you share a code snippet on how you create the OkHttpClient? Do you manually add our interceptor / event listeners similar to this snippet?
```kotlin
val client = OkHttpClient.Builder()
  .addInterceptor(SentryOkHttpInterceptor())
  .eventListener(SentryOkHttpEventListener())
  .build()
```

**Comment by mrzbn** [comment]
@markushi I am using plugin to auto-install dependencies. I am not manually adding the "SentryOkHttpInterceptor" and "SentryOkHttpEventListener".

**Comment by stefanosiano** [comment]
hi @mrzbn 
this issue should have been fixed in SDK version 7.3.0.
Please, add a comment here if that's not the case
